### DISCUSSION TITLE
Saving locally

### DISCUSSION THREAD

**Discussion by e-vissch** [coder/code-server#6150] [Q&A]
I'm using the code-server app with ssh tunnels to an HPC, i.e the code-server is hosted remotely but the app is installed locally. I was hoping to save some summary plots locally. I know how to copy saved plots via rsync etc. 
I also saw there seems to be a way to do this manually on the app by right-clicking a file and downloading.
I was wondering if there was any automatic way to save a file locally from within an (e.g python) file when it runs?

**Comment by code-asher** [comment]
You can edit local files via the local app by selecting "show 
local" when opening a file (only if the browser supports it, like 
Chromium) but I do not think there is a way to make anything 
running in the remote save files locally.
